PhysicsTools/NanoAOD/python/nano_cff.py

```diff
@@ -266,7 +264,12 @@ def nanoAOD_runMETfixEE2017(process,isData):
     process.nanoSequenceCommon.insert(process.nanoSequenceCommon.index(jetSequence),process.fullPatMetSequenceFixEE2017)
 
 def nanoAOD_customizeCommon(process):
-#    makePuppiesFromMiniAOD(process,True) # call this here as it calls switchOnVIDPhotonIdProducer
+    makePuppiesFromMiniAOD(process,True) # call this here as it calls switchOnVIDPhotonIdProducer
+    process.puppiNoLep.useExistingWeights = True
+    process.puppi.useExistingWeights = True
+    for modifier in run2_nanoAOD_106Xv1:
```

Comment:
Sorry to come back to this once more:
Rethinking this once more, I believe even for run2_nanoAOD_106Xv1 we can use existing weights, as the new puppi tune is already applied at UL-re-Miniaod, so NanoAOD needs to do nothing on top of UL-re-MiniAOD.

Reply:
I summary, this PR should not touch the PUPPI weights/tune at all for UL, because this is taken care of in MiniAOD already.

Reply:
So I'd suggest to drop these three lines.

Reply:
Done with the latest commit, modified such that the useExistingWeights = True for all cases such that the existing weights from miniAOD is used for all cases

Reply:
The run2_nanoAOD_106Xv1 labels the ongoing-miniAOD-UL not for the UL-re-MiniAOD

So all in all we will have:
* EOY(94X and 102X) and UL-ongoing(106Xv1) with the old V11 puppi.
* the nano make after the 11_2 and the ones coming from the re-miniAOD-with10_6 will have the V15

Reply:
Discussing with @mariadalfonso, we may potentially have a NanoAOD run2_nanoAOD_106Xv1 before the re-MiniAOD. For this specific case we would want to re-run PUPPI v15.
@saghosh Could you undo 37c4c6fe7d9495cb68740ee0bfa3846b735c45cc, and re-run PUPPI for "run2_nanoAOD_106Xv1"?

Reply:
So we will have:


    EOY(94X and 102X) with the old V11 puppi.
    UL-ongoing(106Xv1) and the nano make after the 11_2 and the ones coming from the re-miniAOD-with10_6 will have the V15

Reply:
For the record, what we want is, for "run2_nanoAOD_106Xv1" rerun Type-1 PUPPI MET and rerun slimmedJetsAK8 (not sure if this PR is doing this).

Reply:
I have added the lines back following your suggestions.